### PR TITLE
114-8fd page and cards

### PR DESCRIPTION
**Pull request by soringifei**
tasks mock page

### CODE REVIEW

**Review by AleBarboni — COMMENTED** [review]
remove white space

**Review by AleBarboni — COMMENTED** [review]
remove white space

File: src/pages/TasksMockPage.jsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
+import React, { useState } from "react";
+
```
Comment (AleBarboni) [inline src/pages/TasksMockPage.jsx:2]: remove white space

File: src/pages/TasksMockPage.jsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+		status: TaskStatus.COMPLETED,
+	},
+];
+
+const TasksMockPage = () => {
+	const [data, setData] = React.useState(mockCards);
+	const [tabIndex, setTabIndex] = React.useState(0);
+	const [status, setStatus] = React.useState(null);
+	const handleChange = (event, newValue) => {
+		console.log("newValue", newValue);
```
Comment (cezarmocanu) [inline src/pages/TasksMockPage.jsx:107]: Remvoe comment

**Review by cezarmocanu — APPROVED** [review]

**Review by AleBarboni — APPROVED** [review]

**Review by MariusStan2001 — APPROVED** [review]

**Pull request merged**